migration/migration.go

```diff
@@ -206,52 +206,32 @@ func PopulateCommonTypes(ctx context.Context, db *gorm.DB, witr models.WorkItemT
 	// q := `ALTER TABLE "tracker_queries" ADD CONSTRAINT "tracker_fk" FOREIGN KEY ("tracker") REFERENCES "trackers" ON DELETE CASCADE`
 	// db.Exec(q)
 
-	if err := createOrUpdateSystemUserstory(ctx, witr, db); err != nil {
+	if err := createOrUpdateSystemPlannerItem(ctx, witr, db); err != nil {
 		return err
 	}
-	if err := createOrUpdateSystemValueProposition(ctx, witr, db); err != nil {
+	if err := createOrUpdateCommon(models.SystemUserStory, ctx, witr, db); err != nil {
 		return err
 	}
-	if err := createOrUpdateSystemFundamental(ctx, witr, db); err != nil {
+	if err := createOrUpdateCommon(models.SystemValueProposition, ctx, witr, db); err != nil {
 		return err
 	}
-	if err := createOrUpdateSystemExperience(ctx, witr, db); err != nil {
+	if err := createOrUpdateCommon(models.SystemFundamental, ctx, witr, db); err != nil {
 		return err
 	}
-	if err := createOrUpdateSystemFeature(ctx, witr, db); err != nil {
+	if err := createOrUpdateCommon(models.SystemExperience, ctx, witr, db); err != nil {
```

Comment:
Perhaps "createOrUpdatePlannerItemType"? Because there are no function overloading in Go, I try to make function names more unique than in other languages.